e2e/crm4/scenarios/scenario-2.spec.js

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from '@playwright/test';
+import { YourApplicationsPage, IsThisPrisonLawPage, ApplyingTotalAmountPage } from '../pages/provider';
+
+import { authenticateAsProvider } from '../../../helpers';
+
+test.describe('CRM4 - Scenario 2', () => {
+	let page;
+	test.describe.configure({ mode: 'serial' });
+	test.beforeAll(async ({ browser }) => {
+		page = await browser.newPage();
+	});
+	test.afterAll(async () => {
+		await page.close();
+	});
+
+	test('Provider - Start a new application that is a Prison Law Matter and less than £500', async () => {
+		await authenticateAsProvider(page);
+		await test.step('Starting a new application', async () => {
+			const yourApplicationsPage = new YourApplicationsPage(page);
+			// Actions
+			await yourApplicationsPage.goto();
+			// Expectations
+			await expect(page.getByRole('button', { name: 'Make a new application' })).toBeVisible();
+			await page.getByRole('button', { name: 'Make a new application' }).click();
+		});
```

Comment:
so a couple of things...
1. the provider and caseworker sides are still 2 tests, which introduces a dependency - we can remove this by making it just one test - yes it would be a long test but that is fine in this case
2. we currently have two layers of abstraction - pages and the tests themselves, I think we could improve the readibility and reuse of the tests by add a 3rd layer in between the existing 2 (already done for `authenticateAsProvider`)

so

```
                await test.step('Starting a new application', async () => {
			const yourApplicationsPage = new YourApplicationsPage(page);
			// Actions
			await yourApplicationsPage.goto();
			// Expectations
			await expect(page.getByRole('button', { name: 'Make a new application' })).toBeVisible();
			await page.getByRole('button', { name: 'Make a new application' }).click();
		});
```
would become:
```
await startANewApplication(page)
```

Reply:
or potentially:
```
await provider.startANewApplication(page)
```
thus namespacing the 3rd layer of actions.. :)